### DISCUSSION TITLE
无法拖入文件夹进行批量识别

### DISCUSSION THREAD

**Discussion by ffmpegzhou** [hiroi-sora/Umi-OCR#670] [General]
拖入任何文件夹都不行，显示这个图标。Win11系统
![315f1d51128543c81473c96a7f39ee9](https://github.com/user-attachments/assets/3bbb5bf1-423f-48e0-b594-56005a208e3f)

**Comment by hiroi-sora** [comment]
应该是系统限制，类似：

https://www.cnblogs.com/lijin-note/p/16542536.html
